packages/mgt-chat/src/statefulClient/GraphNotificationUserClient.ts

```diff
@@ -248,7 +248,7 @@ export class GraphNotificationUserClient {
 
           if (diff <= appSettings.renewalThreshold) {
             this.renewalCount++;
-            log(`Renewing Graph subscription. RenewalCount: ${this.renewalCount}.`);
+            log(`Renewing Graph subscription for ChatList. RenewalCount: ${this.renewalCount}.`);
```

Comment:
Is this only for ChatList and not Chat?

Reply:
There are 2 different files, this one is just for ChatList.